ChangeLog

```diff
@@ -42,6 +42,10 @@ Change log for the astroid package (used to be astng)
 
      Close #437, #447, #313, PyCQA/pylint#1642, PyCQA/pylint#1805, PyCQA/pylint#1854, PyCQA/pylint#1452
 
+  * Infernece now understands the 'isinstance' builtin
```

Comment:
A typo here. Also the following line has a weird indentation, maybe it was indented with tabs instead of spaces?